WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -95,7 +95,11 @@ private extension AddAttributeOptionsViewController {
 
     func createUpdateIndicatorButton() -> UIBarButtonItem {
         let indicator = UIActivityIndicatorView(style: .medium)
-        indicator.color = .primaryButtonTitle
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.largeTitles) {
+            indicator.color = .accent
```

Comment:
nit: the accent color makes the activity indicator look actionable, not sure if one of the gray colors would be good here? (cc @woocommerce/mobile-designers) If we decide on a color, we can add this "navigation bar activity indicator color" to `UIColor+SemanticColors.swift` for reuse in other screens

<img src="https://user-images.githubusercontent.com/1945542/109934702-c77c7e80-7d07-11eb-997f-890e7f9b1df3.png" width="300" />

Reply:
Oh that makes sense, will make that change while @woocommerce/mobile-designers give their opinion!

Reply:
I'm going to merge it but will address the color change after the design team input!